### ISSUE TITLE
Mirroring 

### ISSUE THREAD

**Issue by RealFakeGitter** [futo-org/fcast#4]
One perc of other casting services (like Chromecast) is that they offer you to fully mirror your screen , which as a result unlocks the potential to cast any and every app and site regardless if it's built in or not. Fcast works great with grayjay but if I wanted to stream local videos , or videos from some site or another app I'm now back to being forced to using Chromecast or some other closed source service. Of course while it would be wonderful if every single app that supports casting would built in fcast its unlikely to happen. So it would be so beautiful if Fcast could at least do full screen mirroring as opposed to being limited to only grayjay

**Comment by PGQT** [comment]
screencasting would be incredible. It is definitely a strong point in favor of chromecast and airplay.

**Comment by obakha** [comment]
Any updates in this? Fcast could really use this feature, especially that it has a good potential to replace other non-foss apps.

**Comment by MAlba124** [comment]
We've now released our desktop sender (https://github.com/futo-org/fcast/releases/tag/sender-0.0.1-beta) which has a screen mirroring feature.

**Comment by MAlba124** [comment]
Here is a build of the android sender with screen mirroring. It's not the best quality but I'd appreciate if anyone would share their experience running it on different hardware devices.

https://dl.fcast.org/sender/android/fcast-sender-0.1.0-aarch64.apk (I only built it for arm64 now but if there's interest in other architectures let me know!)

**Comment by ZandercraftGames** [comment]
> Here is a build of the android sender with screen mirroring. It's not the best quality but I'd appreciate if anyone would share their experience running it on different hardware devices.
> 
> https://dl.fcast.org/sender/android/fcast-sender-0.1.0-aarch64.apk (I only built it for arm64 now but if there's interest in other architectures let me know!)

Seems to work great! My only complaint is that it seems to have a lot more compression artifacts showing up on the receiver than with other sender applications.

Edit: Like A LOT of artifacts that stick around persistently even after the screen has moved onto something else.

<img width="1197" height="2545" alt="Image" src="https://github.com/user-attachments/assets/02c5e52f-68ca-4e88-9843-8f7caa9910d9" />

**Comment by MAlba124** [comment]
> Like A LOT of artifacts that stick around persistently even after the screen has moved onto something else.

Yeah, I'm aware of it. I haven't pinpointed the reason for it yet.

**Comment by RealFakeGitter** [comment]
> Here is a build of the android sender with screen mirroring. It's not the best quality but I'd appreciate if anyone would share their experience running it on different hardware devices.
> 
> https://dl.fcast.org/sender/android/fcast-sender-0.1.0-aarch64.apk (I only built it for arm64 now but if there's interest in other architectures let me know!)

Well I can tell you at least on a android Head unit it does not work lol while it connected from my phone side on the receiving end it says "source error, response code 404. I'll try it out on my smart TV later. But that's at least some starting feedback for you.
" 
My sending device is pixel 8 Pro running calyx OS Android 16.

<img width="3060" height="4080" alt="Image" src="https://github.com/user-attachments/assets/9c8d1ccd-210c-4c93-94a6-35e554289f53" />

Update just tried it on my fire tv. No error but I'm getting a totally black screen
